--- old
+++ new
@@ -2,0 +2,44 @@
+README OhMyTermuxScript
+OhMyTermuxScript 🧊
+A collection of scripts for Termux.
+
+Warning
+This program is in Alpha version (see Update log for more information.
+
+ 
+
+🧊 Complete installation
+Important
+To use a beautiful command line script execution interface, gum can be used with -g or --gum parameter added to the execution command.
+
+From a clean install of Termux, paste the following code
+pkg update -y && pkg install git -y
+git clone https://github.com/GiGiDKR/OhMyTermuxScript.git
+cd OhMyTermuxScript && chmod +x ./*.sh
+Type ls to display the list of scripts, then type
+./script_name.sh 
+ 
+
+🧊 Run a specific script
+Electron Node.js
+curl -sL https://raw.githubusercontent.com/GiGiDKR/OhMyTermuxScript/main/electron.sh -o electron.sh && chmod +x electron.sh && ./electron.sh --gum
+Create Symbolic Link to external user directories
+curl -sL https://raw.githubusercontent.com/GiGiDKR/OhMyTermuxScript/main/usersymlink.sh -o usersymlink.sh && chmod +x usersymlink.sh && ./usersymlink.sh --gum
+🧊 Additional parameters
+Uninstall
+./script_name.sh -u
+./script_name.sh --uninstall
+Using gum to magnify the command line interface
+./script_name.sh -g
+./script_name.sh --gum
+Tip
+These settings can be combined.
+
+💻 Update log
+Version 0.1.0 : Initial upload
+Version 0.2.0 : OhMyTermux integration
+⏺️ To do
+Integrate the ability to use gum into scripts
+Integrate uninstallation with a run command parameter
+Update README.md to include script-specific commands
+Upload other scripts after testing